apps/nrm/behaviours/types-of-exploitation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+
+  /**
+   * Check if at least one address field has been populated
+   * @param {*} req - request object
+   */
+  // eslint-disable-next-line complexity
+  isAtLeastOneOptionSelected(req) {
+    if (
+      req.form.values['types-of-exploitation-forced-to-work'] ||
+      req.form.values['types-of-exploitation-wages-taken'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-fraud'] ||
+      req.form.values['types-of-exploitation-prostitution'] ||
+      req.form.values['types-of-exploitation-child-exploitation'] ||
+      req.form.values['types-of-exploitation-taken-somewhere'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-crime'] ||
+      req.form.values['types-of-exploitation-organs-removed'] ||
+      req.form.values['types-of-exploitation-unpaid-household-work'] ||
+      req.form.values['types-of-exploitation-other']
+    ) {
+        return true;
+      }
+
+      return false;
+  }
+
+  validate(req, res, next) {
+    if (!this.isAtLeastOneOptionSelected(req)) {
+      next({
+        'types-of-exploitation-forced-to-work': new this.ValidationError('types-of-exploitation-forced-to-work', {
+          key: 'types-of-exploitation-forced-to-work',
+          type: 'undefined'
```

Comment:
would it be more appropriate to have a more descriptive name then undefined in relation to this `undefined-exploitation`

Reply:
Done.